Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -508,7 +499,7 @@ public class JavaScriptLifter: Lifter {
                 } else {
                     w.emit("\(decl(v)) = \(expression);")
                     if options.contains(.dumpTypes) {
-                        w.emitComment("\(v) = \(interpreter.type(of: v))")
+                        w.emitComment("\(v) = \(program.types.getType(of: v, at: instr.index))")
```

Comment:
I think it would be nice to have a convenience accessor: `Program.type(of: Variable, at: Int)`, wdyt?

Reply:
yeah and it can do some additional work as we talked before. Like checking if variable is existent at requested instruction and return `.nothing` if it is not (instead of .unknown).

But still I would like to do it separately, if not PR at least separate commit.

Reply:
Cool, yeah either is fine for me, whatever you prefer :)